examples/taskfile.yml

```diff
@@ -12,10 +12,11 @@ includes:
     with-gorm: ./with-gorm/taskfile.yml
     with-multi-content-types: ./with-multi-content-types/taskfile.yml
     with-swagger-ui: ./with-swagger-ui/taskfile.yml
+    with-validation: ./with-validation/taskfile.yml
 
 tasks:
     run:
       cmds:
-          - for: [basic, with-azure-functions, with-cache, with-cli, with-cobra, with-custom-functions, with-custom-type-mapping, with-gorm, with-multi-content-types, with-swagger-ui]
+          - for: [basic, with-azure-functions, with-cache, with-cli, with-cobra, with-custom-functions, with-custom-type-mapping, with-gorm, with-multi-content-types, with-swagger-ui, with-validation]
```

Comment:
Thing to test, is this supported by yaml lists (I.e. newline and hyphen) to make it easier to read?

Reply:
I have updated it to use yaml list, soo much better